fem/fespace.hpp

```diff
@@ -338,6 +379,14 @@ class FiniteElementSpace
    const QuadratureInterpolator *GetQuadratureInterpolator(
       const QuadratureSpace &qs) const;
 
+   /** @brief Return a FaceQuadratureInterpolator that interpolates E-vectors to
+       quadrature point values and/or derivatives (Q-vectors). */
+   const FaceQuadratureInterpolator *GetFaceQuadratureInterpolator(
+      const IntegrationRule &ir, FaceType type) const;
+
+   const FaceQuadratureInterpolator *GetFaceQuadratureInterpolator(
+      const QuadratureSpace &qs, FaceType type) const;
```

Comment:
I don't think `QuadratureSpace` is the right type here. The method that is used by the returned `FaceQuadratureInterpolator` (in its `Mult` method) is `QuadratureSpace::GetElementIntRule` which will return a quadrature for an element, not a face.

For now I think it is OK to remove this version of the `GetFaceQuadratureInterpolator` method. We can introduce it later when we add support for meshes with different types of faces and/or variable-order spaces where we want to vary the face quadratures.